test/Interop/Cxx/value-witness-table/copy-basic.swift

```diff
@@ -0,0 +1,51 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
```

Comment:
Please rename the test to follow the name of the C++ module.

`basic-execution.swift`